cluster-autoscaler/processors/nodeinfosprovider/gce_node_info_provider.go

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
>Isn't providing annotations something that should be a part of MixedTemplateNodeInfoProvider?
I wasn't sure about it, because I don't know how other cloud providers implements TemplateNodeInfo() and use Annotations. Since that I decided to add new processor is a safe option. I named it as a GceNodeInfoProvider because it is used only for GCE cloud provider.